README.md

```diff
@@ -49,6 +49,7 @@ They must be placed inside the same folder as the entity definition and must be
 
 - Golden metrics: `golden_metrics.yml`
 - Summary metrics: `summary_metrics.yml `
+- Relationship Definitions: `relationships.yml `
```

Comment:
Sorry i misplaced this comment elsewhere.  I think the relationship synthesis defs. are better suited for `definitions.yml`